test/mock_data.py

```diff
@@ -494,6 +494,7 @@
     **CATALOGUE_ITEM_DATA_REQUIRED_VALUES_ONLY,
     "catalogue_category_id": str(ObjectId()),
     "manufacturer_id": str(ObjectId()),
+    "number_of_spares": None,
 }
```

Comment:
```suggestion
    **CATALOGUE_ITEM_DATA_REQUIRED_VALUES_ONLY,
    "catalogue_category_id": str(ObjectId()),
    "manufacturer_id": str(ObjectId()),
}
```
It's optional, so it shouldn't be included in the `REQUIRED_VALUES_ONLY`?

Reply:
The reason it is like this is because the `CatalogueItemBase` doesn't have it set to `None` automatically. The idea being that with the actual calculation, I never want it to default to anything - it should only be None if the calculation itself returns it, but forgetting to set it will cause an error and prompt a developer to use the calculation to populate it instead.  The others are set to None because if you forget to set it, its fine.